.github/workflows/docker_build.yml

```
@@ -1,31 +1,53 @@
-##   docker-verify:
-##     runs-on: ubuntu-latest
-##     steps:
-##       - name: Checkout repository
-##         uses: actions/checkout@v4
-#
-##       - name: Set up Docker Buildx
-##         uses: docker/setup-buildx-action@v3
-#
-##       - name: Verify Docker Compose Build
-##         run: docker compose -f docker-compose.yml build
+name: CI
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)